test/sanitizer_suppressions/tsan

```diff
@@ -25,8 +25,8 @@ race:WalletBatch::WriteHDChain
 race:BerkeleyBatch
 race:BerkeleyDatabase
 race:DatabaseBatch
+race:SendZmqMessage
```

Comment:
Looking at the [log](https://github.com/bitcoin/bitcoin/issues/20618#issue-761612410), I'd suggest to be more specific:
```suggestion
race:CZMQAbstractPublishNotifier::SendZmqMessage
```

Reply:
Thanks, done